UniConnectApp/app/src/main/java/com/example/uniconnect/MainActivity.kt

```diff
@@ -25,16 +25,14 @@ import com.google.firebase.auth.FirebaseUser
 
 class MainActivity : ComponentActivity() {
     private val viewModel: MainViewModel by viewModel<MainViewModel>()
-    //private var inTitle : String = ""
-    //private var inDescription : String = ""
```

Comment:
Good to remove.
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.